--- src/sysinit_11xx.c.OLD	2013-11-16 14:08:32.000000000 +0200
+++ src/sysinit_11xx.c	2013-10-14 10:11:53.000000000 +0300
@@ -76,7 +76,7 @@
 	Chip_Clock_SetSysClockDiv(1);
 
 	/* Setup FLASH access to 3 clocks */
-	Chip_FMC_SetFLASHAccess(FLASHTIM_50MHZ_CPU);
+//	Chip_FMC_SetFLASHAccess(FLASHTIM_50MHZ_CPU);
 
 	/* Set main clock source to the system PLL. This will drive 48MHz
 	   for the main clock and 48MHz for the system clock */
--- inc/pmu_11xx.h.OLD	2013-11-16 14:08:32.000000000 +0200
+++ inc/pmu_11xx.h	2013-11-16 14:03:28.000000000 +0200
@@ -46,7 +46,7 @@
 
 #if defined(CHIP_LPC11AXX) || defined(CHIP_LPC11CXX) || defined(CHIP_LPC11EXX) || defined(CHIP_LPC11UXX) || defined(CHIP_LPC1125)
 #if defined(CHIP_LPC1125)
-#error "LPC1125 support for the PMU driver is not ready
+#error "LPC1125 support for the PMU driver is not ready"
 #endif
  
 /**
