--- src/system_LPC13xx.c.old	2013-11-18 11:48:33.000000000 +0200
+++ src/system_LPC13xx.c	2013-11-21 19:55:59.000000000 +0200
@@ -148,7 +148,7 @@
 // ******** Code Red *********
 // * Changed USBCLK_SETUP to 1
 // ***************************
-#define USBCLK_SETUP          1
+#define USBCLK_SETUP          0
 #define USBPLL_SETUP          1
 #define USBPLLCLKSEL_Val      0x00000001
 #define USBPLLCTRL_Val        0x00000003
